cmd/deploy/deploy.go

```diff
@@ -474,27 +474,36 @@ func (dc *DeployCommand) deploy(ctx context.Context, opts *Options) error {
 
 		// deploy compose if any
 		if opts.Manifest.Deploy.ComposeSection != nil {
+			oktetoLog.SetStage("Deploying compose")
 			if err := dc.deployStack(ctx, opts); err != nil {
+				oktetoLog.AddToBuffer(oktetoLog.ErrorLevel, "error deploying compose: %s", err.Error())
 				exit <- err
 				return
 			}
+			oktetoLog.SetStage("")
 		}
 
 		// deploy endpoits if any
 		if opts.Manifest.Deploy.Endpoints != nil {
+			oktetoLog.SetStage("Endpoints configuration")
 			if err := dc.deployEndpoints(ctx, opts); err != nil {
+				oktetoLog.AddToBuffer(oktetoLog.ErrorLevel, "error deploying divert: %s", err.Error())
```

Comment:
```suggestion
				oktetoLog.AddToBuffer(oktetoLog.ErrorLevel, "error deploying endpoints: %s", err.Error())
```

Reply:
Endpoints are not deployed right? Maybe "error creating endpoints" or "generating endpoints " could be more accurate?